templates/admin-page.php

```diff
@@ -22,6 +22,10 @@
 					</label>
 				</h2>
 
+				<p>
+					<?php esc_html_e( 'Select a plugin to check best practices in several categories and security issues.', 'plugin-check' ); ?>
```

Comment:
This reads a bit confusing IMO. I wouldn't call out "security issues" explicitly since it's one of the "several categories" anyway. Maybe better something like this?
```suggestion
					<?php esc_html_e( 'Select a plugin to check it for best practices in several categories.', 'plugin-check' ); ?>
```

Reply:
Yeah, why not.

Reply:
One follow up:
```suggestion
					<?php esc_html_e( 'Select a plugin to check it for best practices in several categories and security issues.', 'plugin-check' ); ?>
```